routes/v1.js

```diff
@@ -731,8 +756,11 @@ api.declare({
   name:       'cancelTask',
   scopes:     [
     [
+      // Legacy scopes
       'queue:cancel-task',
       'assume:scheduler-id:<schedulerId>/<taskGroupId>'
+    ], [
+      'queue:cancel-task:<schedulerId>/<taskGroupId>',
```

Comment:
Better to use `queue:cancel-task:<schedulerId>/<taskGroupId>/<taskId>`. Note, `queue:cancel-task:<schedulerId>/<taskGroupId>/*` would provide existing semantics, but allow finer grained access control if required, when a component should only be allowed to cancel an individual task in a task graph, rather than an arbitrary one. This could otherwise be otherwise open us up for an attack strategy whereby somebody gets access to cancel a "sister" task of a task graph they managed to get access to by virtue of having e.g. extended the task graph with an unrelated job.